superset-frontend/src/CRUD/CollectionTable.tsx

```diff
@@ -67,15 +74,23 @@ const CrudTableWrapper = styled.div<{ stickyHeader?: boolean }>`
     stickyHeader &&
     `
       height: 350px;
-      overflow: auto;
+      overflow-y: auto;
```

Comment:
`overflow: auto` is a shorthand for `overflow-x` and `overflow-y`.

Reply:
I had specify specifically in this case for some reason because just overflow:auto did not work.